specifyweb/frontend/js_src/lib/components/FormEditor/viewSpec.ts

```diff
@@ -313,6 +315,15 @@ const labelSpec = f.store(() =>
   })
 );
 
+const blankSpec = f.store(() =>
```

Comment:
@maxpatiiuk, is this necessary or is using emptySpec sufficient?

Reply:
if the cell is blank, then it's not supposed to have any attributes, right?
so yeah, emptySpec is the right choice at the moment

though, when developing a visual editor, we would have to consider how empty cells and unknown cells should be treated